templates/registration/password_reset_done.html

```diff
@@ -0,0 +1,19 @@
+<!-- templates/registration/password_reset_done.html -->
+{% extends 'base.html' %}
+
+{% block title %}Password reset send{% endblock %}
+
+{% block content %}
+
+
+<div class="row justify-content-center">
+    <div class="col-md-8">
+        <h2 class="text-center">Password reset sent</h2>
+        <p>We’ve emailed you instructions for setting your password, if an account exists with the email you entered. You should receive them shortly.</p>
+        <p>If you don’t receive an email, please make sure you’ve entered the address you registered with, and check your spam folder.</p>
```

Comment:
Users don't register, so they can't make sure of that. I would suggest "please check your spam folder or contact administrator".